README.md

```diff
@@ -163,6 +163,14 @@ Keys:
 
 **Default**: `"- examples"`
 
+If you want to specify individual sketches to be compiled, you can do so by passing a multi-line string (a so called [literal block scalar](https://www.yaml.info/learn/quote.html#literal)), i.e.
+```
+sketch-paths: |
+  - examples/Example-01
+  - examples/Example-03
+  - examples/Example-03
+```
```

Comment:
````suggestion
<details>
  <summary><b>Click for an example</b></summary>
  <p>

```
sketch-paths: |
  - examples/Example-01
  - examples/Example-03
  - examples/Example-03
```

  </p>
</details>
````

This markup will put the example snippet in a collapsible section that is expanded by the reader on demand with a click:

<details>
  <summary><b>Click for an example</b></summary>
  <p>

```yaml
sketch-paths: |
  - examples/Example-01
  - examples/Example-03
  - examples/Example-03
```

  </p>
</details>

The reason I propose this approach is because I think it would be good to add similar examples to the other complex inputs, but such a significant amount of added content can harm the browsability of the readme for those who are looking for a general overview of the action's API.